Rmarkdown/Multiple_Ascending_Dose_PK.Rmd

```diff
@@ -1,573 +1,451 @@
+  select(-TRTACT)
+
+
+#create pk dataset
+pk_data <- pkpd_data %>%
+  filter(CMT==2)
+
+#perform NCA, for additional plots
+NCA = pk_data %>%
+  group_by(ID, DOSE) %>%
+  filter(!is.na(LIDV)) %>%
+  summarize(AUC_0_24 = caTools::trapz(TIME[NOMTIME>0 & NOMTIME<=24], LIDV[NOMTIME>0 & NOMTIME<=24]),
+            Cmax_0_24     = max(LIDV[NOMTIME>0 & NOMTIME<=24]),
+            AUC_tau = caTools::trapz(TIME[NOMTIME>120 & NOMTIME<=144],LIDV[NOMTIME>120 & NOMTIME<=144]),
+            Cmax_tau     = max(LIDV[NOMTIME>120 & NOMTIME<=144]),
+            AUC_acc = AUC_tau / AUC_0_24,
+            Cmax_acc = Cmax_tau / Cmax_0_24,
+            SEX      = SEX[1], #this part just keeps the SEX and WEIGHTB covariates
+            WEIGHTB  = WEIGHTB[1]) %>%
+  gather(PARAM, VALUE,-c(ID, DOSE, SEX, WEIGHTB)) %>%
+  ungroup() %>%
+  mutate(VALUE_NORM = VALUE/DOSE,
+         PROFDAY = ifelse(PARAM %in% c("AUC_0_24", "Cmax_0_24"), 1, 6),
+         DAY_label = paste("Day", PROFDAY))
+
+
+#units and labels
+  time_units_dataset = "hours"
+  time_units_plot    = "days"
+  trtact_label       = "Dose"
+  dose_units         = unique((pkpd_data %>% filter(CMT==1) )$EVENTU) %>% as.character()
+  dose_label         = paste0("Dose (", dose_units,")")
+  conc_units         = unique(pk_data$LIDV_UNIT) %>% as.character()
+  conc_label         = paste0("Concentration (", conc_units, ")")
+  concnorm_label     = paste0("Normalized Concentration (", conc_units,")/", dose_units)
+  AUC_units          = paste0("h.", conc_units)
+                              
+#directories for saving individual graphs
+dirs = list(
+  parent_dir= "Parent_Directory",
+  rscript_dir  = "./",
+  rscript_name = "Example.R",
+  results_dir  = "./",
+  filename_prefix   = "",
+  filename     = "Example.png")
+  
+```
+
+## Provide an overview of the data
+
+Summarize the data in a way that is easy to visualize the general trend of PK over time and between doses. Using summary statistics can be helpful, e.g. Mean +/- SE, or median, 5th & 95th percentiles. Consider either coloring by dose or faceting by dose. Depending on the amount of data one graph may be better than the other.
+
+When looking at summaries of PK over time, there are several things to observe. Note the number of doses and number of time points or sampling schedule. Observe the overall shape of the average profiles. What is the average Cmax per dose? Tmax? Does the elimination phase appear to be parallel across the different doses? Is there separation between the profiles for different doses? Can you make a visual estimate of the number of compartments that would be needed in a PK model?
+
+### Concentration over Time, colored by dose, mean +/- 95% CI
+
+```{r , cache = TRUE, echo = TRUE, warning=FALSE, message=FALSE, fig.width = 10, fig.height = 5}
+
+gg <- ggplot(data = pk_data, 
+             aes(x = NOMTIME, y = LIDV, 
+                 group= interaction(DOSE,CYCLE), color = TRTACT_high2low)) 

```

Comment:
add space between "group" and "=".  I'm maybe too anal...

Reply:
Added spaces around all "="